docs/index.md

```diff
@@ -20,6 +20,7 @@ Testcontainers makes it easy to launch useful Docker containers for the duration
 
  * [ZeroTurnaround](https://zeroturnaround.com) - Testing of the Java Agents, micro-services, Selenium browser automation
  * [Zipkin](http://zipkin.io) - MySQL and Cassandra testing
+ * [Apache Camel](https://camel.apache.org) - Testing Camel agains native service such as Consul, Etcd and so on.
```

Comment:
😍 nice! Just a small note: the list is in the chronological order, could you please change it? :)